app/views/layouts/metadata_presenter/application.html.erb

```diff
@@ -30,7 +30,6 @@
     </div>
     <%= render template: 'metadata_presenter/footer/footer' %>
 
-    <%= javascript_pack_tag 'application' %>
     <%= javascript_pack_tag 'govuk' %>
```

Comment:
Are we certain the application js is not needed?  I understand that it *shoudln't* be needed, but just want to make sure we don't break forms in production...

Reply:
Yes, I believe so.

Application.js basically did this:
- Pull in jQuery stuff
- Pull in Showdown (markdown editing support)
- Pull in Editor index.js for Editor JS support
- Pull in Editor CSS

None of that we should need since it is all Editor specific and, the only things we want in Runner are the GDS CSS